package.json

```diff
@@ -20,79 +20,86 @@
     "eslint-loader": "^1.1.1",
     "eslint-plugin-mocha": "^1.1.0",
-    "eslint-plugin-react": "3.10.0",
-    "history": "^1.12.3",
-    "express": "^4.13.1",
-    "extract-text-webpack-plugin": "^0.8.1",
-    "file-loader": "^0.8.4",
-    "image-webpack-loader": "^1.6.1",
-    "imagemin": "^3.2.0",
-    "isparta-loader": "^0.2.0",
-    "karma": "^0.13.9",
-    "karma-chrome-launcher": "^0.2.0",
-    "karma-cli": "^0.1.0",
-    "karma-coverage": "^0.5.0",
-    "karma-firefox-launcher": "^0.1.6",
-    "karma-mocha": "^0.2.0",
-    "karma-mocha-reporter": "^1.1.1",
-    "karma-sinon-chai": "^1.0.0",
-    "karma-sourcemap-loader": "^0.3.5",
+    "eslint-plugin-react": "3.13.1",
+    "express": "^4.13.3",
+    "extract-text-webpack-plugin": "^0.9.1",
+    "file-loader": "^0.8.5",
+    "history": "^1.17.0",
+    "image-webpack-loader": "^1.6.2",
+    "imagemin": "^4.0.0",
+    "invariant": "^2.2.0",
+    "isparta-loader": "^2.0.0",
+    "karma": "^0.13.16",
+    "karma-chrome-launcher": "^0.2.2",
+    "karma-cli": "^0.1.2",
+    "karma-coverage": "^0.5.3",
+    "karma-firefox-launcher": "^0.1.7",
+    "karma-mocha": "^0.2.1",
+    "karma-mocha-reporter": "^1.1.3",
+    "karma-sinon-chai": "^1.1.0",
+    "karma-sourcemap-loader": "^0.3.6",
     "karma-webpack": "^1.7.0",
-    "lost": "^6.3.0",
+    "lost": "^6.6.2",
     "matthewmueller-react-remarkable": "^1.1.2",
-    "mocha": "^2.2.5",
-    "node-libs-browser": "^0.5.2",
+    "mocha": "^2.3.4",
+    "node-libs-browser": "^0.5.3",
     "normalize.css": "^3.0.3",
-    "object-assign": "^2.0.0",
-    "postcss-calc": "^4.0.1",
-    "postcss-color-function": "^1.2.0",
-    "postcss-custom-properties": "^3.3.0",
-    "postcss-loader": "^0.6.0",
-    "postcss-simple-vars": "^0.3.0",
+    "object-assign": "^4.0.1",
+    "postcss-calc": "^5.0.0",
+    "postcss-color-function": "^2.0.0",
+    "postcss-custom-properties": "^5.0.0",
+    "postcss-loader": "^0.8.0",
+    "postcss-simple-vars": "^1.2.0",
     "raw-loader": "^0.5.1",
-    "react": "^0.13.3",
-    "react-transform-catch-errors": "^0.1.1",
+    "react": "^0.14.3",
+    "react-dom": "^0.14.3",
+    "react-transform-catch-errors": "^1.0.1",
     "react-transform-hmr": "^1.0.1",
-    "redbox-react": "^1.0.1",
-    "rimraf": "^2.4.2",
-    "sinon": "^1.16.1",
+    "redbox-react": "^1.2.0",
+    "rimraf": "^2.5.0",
+    "sinon": "^1.17.2",
     "sinon-chai": "^2.8.0",
-    "style-loader": "^0.12.3",
-    "svgo": "^0.5.2",
+    "style-loader": "^0.13.0",
+    "svgo": "^0.6.1",
     "svgo-loader": "^1.1.0",
-    "url-loader": "^0.5.6",
-    "webpack": "^1.9.10",
-    "webpack-dev-middleware": "^1.2.0",
+    "url-loader": "^0.5.7",
+    "webpack": "^1.12.9",
+    "webpack-dev-middleware": "^1.4.0",
     "webpack-fail-plugin": "^1.0.2",
-    "webpack-hot-middleware": "^2.0.0",
+    "webpack-hot-middleware": "^2.6.0",
     "webpack-postcss-tools": "^1.1.1"
   },
   "peerDependencies": {
-    "react": "^0.13.0"
+    "react": "^0.14.0"
   }
 }
```

Comment:
I used [npm-check-updates](https://www.npmjs.com/package/npm-check-updates) to update all deps to latest.